sdk/storage/AzureStorageBlob/Source/Options/OptionsModels.swift

```diff
@@ -154,7 +154,7 @@ public struct CustomerProvidedEncryptionKey: Codable, Equatable {
     }
 
     /// Specifies the algorithm to use when encrypting data using the given key. Must be AES256.
-    public let algorithm = "AES256"
+    var algorithm = "AES256"
```

Comment:
Unrelated change?

Reply:
It was to get rid of an XCode warning.

Reply:
Hmm, odd. Is it just because it's unused? If so I guess maybe just delete it?